graphql_compiler/tests/test_helpers.py

```diff
@@ -842,12 +846,22 @@ def get_sqlalchemy_schema_info(dialect: str = "mssql") -> SQLAlchemySchemaInfo:
 
     join_descriptors: Dict[str, Dict[str, DirectJoinDescriptor]] = {}
     for edge in edges:
-        join_descriptors.setdefault(edge["from_table"], {})[
-            "out_{}".format(edge["name"])
-        ] = DirectJoinDescriptor(edge["from_column"], edge["to_column"])
-        join_descriptors.setdefault(edge["to_table"], {})[
-            "in_{}".format(edge["name"])
-        ] = DirectJoinDescriptor(edge["to_column"], edge["from_column"])
+        if "column_pairs" in edge:
+            join_descriptors.setdefault(edge["from_table"], {})[
+                "out_{}".format(edge["name"])
+            ] = CompositeJoinDescriptor(edge["column_pairs"])
+            join_descriptors.setdefault(edge["to_table"], {})[
+                "in_{}".format(edge["name"])
+            ] = CompositeJoinDescriptor(
+                {(to_column, from_column) for from_column, to_column in edge["column_pairs"]}
```

Comment:
Just to check, should this be `(to_column, from_column)` or `(from_column, to_column)`?

Reply:
It's correct. This is the edge in the backwards direction. I'll add a comment to explain here.